lib/containers/common.pm

```diff
@@ -77,7 +77,7 @@ sub install_docker_when_needed {
             }
             else {
                 if (is_sle() && script_run("SUSEConnect --status-text | grep Containers") != 0) {
-                    add_suseconnect_product("sle-module-containers");
+                    is_sle('<15') ? add_suseconnect_product("sle-module-containers", 12) : add_suseconnect_product("sle-module-containers");
```

Comment:
Too much nesting for my taste. Not necessary to rework it now, but I would like to replace ternary for `if-else` as it would be more readable in this particular case.

Reply:
can we merge it now and you work on this afterwards? It blocks quite some updates

Reply:
ah, was not aware of that. Sure! Sorry!